libs/compute_api/src/spec.rs

```diff
@@ -27,6 +27,12 @@ pub struct ComputeSpec {
     pub cluster: Cluster,
     pub delta_operations: Option<Vec<DeltaOp>>,
 
+    /// An optinal hint that can be passed to speed up startup time if we know
+    /// that no mutations (like role creation, database creation) need to be
+    /// done on the actual database to start.
+    #[serde(default)] // Default false
+    pub already_configured: bool,
```

Comment:
LGTM, I'm still thinking about making this a enum as I mentioned before. E.g. `config_mode: [full, minimal]`. Yes, now it's still boolean logic, but it's more extensible, i.e. if we will ever need one-time actions like catalog backpatching. And it's hard to modify spec format because of backward compatibility

Yet, we can add a enum later and switch to it if needed, so up to you

Reply:
My thoughts on it https://github.com/neondatabase/cloud/pull/5377#discussion_r1226628774

Reply:
renamed it to `skip_pg_catalog_updates` as discussed